### PR TITLE
[New Rule] AWS RDS Instance Modified or Deleted

### LINKED ISSUE #1264: [New Rule] AWS RDS Instance Modified or Deleted
## Description
AWS RDS Instance Modified or Deleted

### Target indexes

`filebeat-*, logs-aws*`

### Platforms

AWS Cloudtrail

### Query

```
event.dataset:aws.cloudtrail and 
event.provider:rds.amazonaws.com and 
event.action: 
    (
    ModifyDBInstance or
    DeleteDBInstance
    ) and 
event.outcome:success
```

### **False Positives**
- 
- 

### MITRE

| Tactic | Technique ID | Technique Name | Sub-Technique Name |
| ------ | ------------ | -------------- | ------------------ |
|        |              |                |                    |
|        |              |                |                    |

### References
- https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DeleteDBInstance.html
- https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_ModifyDBInstance.html

### PR DESCRIPTION
**Pull request by austinsonger**
## Issues

Resolves #1264

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@
 /*.yml
 /config
 /junit.xml
+/elastic-detection-cli
```
Comment (w0rk3r) [inline .gitignore:11]:
```suggestion
```

File: rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+
```
Comment (w0rk3r) [inline rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml:5]: Missing Integration

File: rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+[rule]
+author = ["Elastic"]
```
Comment (w0rk3r) [inline rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml:7]: Missing you as author?

File: rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+description = """
+Identifies the modification or deletion of an Amazon Relational Database Service (RDS) Aurora database instance.
```
Comment (w0rk3r) [inline rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml:9]: This one could be elaborated on why this should be monitored

**Comment by w0rk3r** [comment]
Closing this one due to lack of activity, feel free to open a new PR if we progress. Thanks!

File: rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml (hunk 4 above)
Comment (austinsonger) [reply rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml:10]: Better Now.

Comment (austinsonger) [reply rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml:10]: For some reason my change isn't updating, probably cause the PR is closed.

https://github.com/austinsonger/detection-rules/blob/AWS-RDS-Instance-Modified-or-Deleted/rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml

**Pull request closed without merging**